plugins/org.locationtech.udig.catalog.db2/src/org/locationtech/udig/catalog/internal/db2/ui/DB2WizardPage.java

```diff
@@ -48,11 +47,12 @@
  * </p>
  * 
  * @author Justin Deoliveira, jdeolive, for Refractions Research, Inc.
- * @author dadler
  * @author Jesse Eichar, jeichar, for Refractions Research, Inc.
  * @author Richard Gould, rgould, for Refractions Research, Inc.
  * @author Adrian Custer, acuster.
  * @since 1.0.1
+ * @author David Adler, dadler, for AdtechGeospatial 
+ * @since 2.0.x 
```

Comment:
David, I guess since 1.0.1 includes 2.0.x for API docs. I'd like to suggest to remove the second one. Or ist there any other intention to add 2.0.x?